recipes-multimedia/alsa-state/alsa-state.bbappend

```
@@ -0,0 +1,22 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}/${MACHINE}:${THISDIR}/${PN}:"
```

@V-SINGH1 For your use-case, following should be enough as you don't have MACHINE specific directories

```suggestion
FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
```